src/lib-sana/Utils/Utils.cpp

```diff
@@ -1,5 +1,52 @@
+#include <random>
+#include <sstream>
+#include <fstream>
+#include <iterator>
+#include "Random.hpp"
 #include "Utils.hpp"
 
 using namespace std;
 
+RandomNumberGenerator randObj;
+mt19937 gen(randObj.GetSeed());
+ranlux24_base fastGen(randObj.GetSeed());
+
+int Utils::RandInt(int low, int high) {
+    uniform_int_distribution <> dis(low,high);
+    return dis(fastGen);
+}
+
+vector < vector<string> > Utils::fileToStringsByLines(const string &fileName) {
```

Comment:
We might want to avoid having this function, any function that needs to read from file makes more sense to directly read from the file.

One more reason is that this function will load the whole file into memory and then after we make a graph out of it will use double amount of memory. In normal cases, does not matter but with big graphs will slow down the initilization.